kolibri/plugins/management/assets/src/vue/modal/index.vue

```diff
@@ -5,10 +5,10 @@
       <div class="modal-wrapper">
         <div class="modal-container">
           <div class="modal-header">
+            <a @click="closeModal" class="close-btn">X</a>
```

Comment:
It would be preferable to use `<button>` element here instead of `<a>`, and add the following inside:
```
<span aria-hidden="true">&times;</span>
<span class="visuallyhidden">Close</span>
```